api/v1beta1/module_types.go

```diff
@@ -65,6 +65,10 @@ type Build struct {
 	// +optional
 	// KanikoParams is used to customize the building process of the image.
 	KanikoParams *KanikoParams `json:"kanikoParams,omitempty"`
+
+	// +optional
+	// Selector describes on which nodes the Job should be created.
+	Selector map[string]string `json:"selector,omitempty"`
```

Comment:
How about a `Sign` selector? What happens if we need to build and sign? We still have the same issue as we had before aren't we? The user will still need a "special node" with some HW in order to sign.

Reply:
@ybettan I will close this comment because based on the community meeting and the notes, this is no longer valid, if so, we can just re-open it again :+1: